test/samples/anyOfMultipleTypes/input.json

```diff
@@ -1,20 +1,24 @@
 {
-	"id": "http://yourdomain.com/schemas/myschema.json",
-	"description": "Example description",
-	"type": "object",
-	"properties": {
-		"first_name": {
-			"anyOf": [
-				{
-					"type": "string"
-				},
-				{
-					"type": "null"
-				}
-			]
-		}
-	},
-	"required": [
-		"first_name"
-	]
-}	
+    "id": "http://yourdomain.com/schemas/myschema.json",
+    "description": "Example description",
+    "allOf": [
```

Comment:
Can you remove all the allOf's that have been added to the test examples, we only need them in one.